pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -177,7 +197,15 @@ assert allPkgconfigDepends != [] -> pkgconfig != null;
 stdenv.mkDerivation ({
   name = "${pname}-${version}";
 
-  outputs = if (args ? outputs) then args.outputs else ([ "out" ] ++ (optional enableSeparateDataOutput "data") ++ (optional enableSeparateDocOutput "doc"));
+  outputs = if (args ? outputs) then args.outputs else
+    ( [ "out" ]
```

Comment:
Note that in rest of nixpkgs, if there is a `bin` output, it's always the first. This convention should probably be followed here.

That way people who write e.g. `${git-annex}/bin/git-annex` in their scripts won't have their scripts broken.